### PR TITLE
VB-3776 Add application creation that capacity has not been exceeded …

### PR DESCRIPTION
**Pull request by AledWEvans**
Added application creation that capacity has not been exceeded, api change to match visip.
Also had a small tidy up side some of the code was not in application mode

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/visit/scheduler/application/ChangeApplicationDto.kt [hunk 1]
```diff
@@ -27,4 +27,6 @@ data class ChangeApplicationDto(
   @Schema(description = "additional support associated with the visit, if null support will not be updated", required = false)
   @Valid
   var visitorSupport: ApplicationSupportDto? = null,
+  @field:NotNull
```
Comment (dhirajshettymoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/visit/scheduler/application/ChangeApplicationDto.kt:30]: @Schema needed for swagger docs?

**Review by dhirajshettymoj — COMMENTED** [review]
1 minor comment added

**Review by dhirajshettymoj — APPROVED** [review]
LGTM

**Review by tpmcgowan — APPROVED** [review]
LGTM

**Pull request merged**